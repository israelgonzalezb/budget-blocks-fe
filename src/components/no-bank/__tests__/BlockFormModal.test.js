
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import BlockFormModal from '../BlockFormModal';


describe('BlockFormModal test', () => {
  it('BlockFormModal should match snapshot', () => {
    const component = renderer.create(<BlockFormModal
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
