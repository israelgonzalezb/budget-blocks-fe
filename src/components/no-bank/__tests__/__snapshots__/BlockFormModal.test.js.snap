// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockFormModal test BlockFormModal should match snapshot 1`] = `null`;
